build-browser.js

```diff
@@ -6,7 +6,7 @@ import babelPresetEnv from '@babel/preset-env';
 import pkg from './package.json';
 
 rollup({
-  entry: 'src/index.js',
+  input: 'src/index.js',
```

Comment:
what's the difference here? `entry` vs. `input`

Reply:
Rollup latest version has deprecated the `entry` property [1479](https://github.com/rollup/rollup/issues/1479)